<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StylePicker</class>
 <widget class="QDialog" name="StylePicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>912</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>880</y>
     <width>166</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>962</width>
     <height>333</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/resources/style.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>349</y>
     <width>960</width>
     <height>176</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/resources/cap.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>531</y>
     <width>963</width>
     <height>345</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/resources/join.png</pixmap>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup1</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup1</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>150</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup1</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_4">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>320</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup1</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>320</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup1</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup1</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_7">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>500</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup2</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_8">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>500</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup2</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_9">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>500</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup2</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_10">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>850</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup3</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_11">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>850</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup3</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_12">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>850</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup3</string>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StylePicker</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StylePicker</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup3"/>
  <buttongroup name="buttonGroup1"/>
  <buttongroup name="buttonGroup2"/>
 </buttongroups>
</ui>
